cmd/ore/aws/gc.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdGC = &cobra.Command{
+		Use:   "gc",
+		Short: "GC resources in AWS",
+		Long:  `Delete GC instances created over the given duration ago`,
+		RunE:  runGC,
+	}
+
+	gcDuration time.Duration
+)
+
+func init() {
+	AWS.AddCommand(cmdGC)
+	cmdGC.Flags().DurationVar(&gcDuration, "duration", 2*time.Hour, "how old resources must be before they're considered garbage")
```

Comment:
Defaulting to two hours seems a bit aggressive.  6?  24?

Reply:
I split the difference and picked 5